packages/lab/stories/interactable-card/interactable-card.stories.tsx

```diff
@@ -0,0 +1,336 @@
+  typeof InteractableCard
+> = (args) => (
+  <InteractableCardGroup>
+    <InteractableCard {...args} value="card" style={{ width: "180px" }}>
+      <StackLayout gap={1}>
+        <StackLayout gap={1} direction="row" align="center">
+          <CreditCardIcon size={2} />
+          <H3>Credit Card</H3>
+        </StackLayout>
+        <Text>Link credit card to your payments account.</Text>
+      </StackLayout>
+    </InteractableCard>
+    <InteractableCard {...args} style={{ width: "180px" }} disabled>
+      <StackLayout gap={1}>
+        <StackLayout gap={1} direction="row" align="center">
+          <BankIcon size={2} />
+          <H3 disabled>Bank wire</H3>
+        </StackLayout>
+        <Text disabled>Link your bank to your payments account.</Text>
+      </StackLayout>
+    </InteractableCard>
+    <InteractableCard {...args} value="crypto" style={{ width: "180px" }}>
+      <StackLayout gap={1}>
+        <StackLayout gap={1} direction="row" align="center">
+          <DiamondIcon size={2} />
+          <H3>Cryptocurrency</H3>
+        </StackLayout>
+        <Text>Link your crypto to your payments account.</Text>
+      </StackLayout>
+    </InteractableCard>
+  </InteractableCardGroup>
+);
+
+export const InteractableCardGroupMultiSelect: StoryFn<
+  typeof InteractableCard
+> = (args) => (
+  <InteractableCardGroup selectionVariant="multiselect">
+    <InteractableCard {...args} value="card" style={{ width: "180px" }}>
+      <StackLayout gap={1}>
+        <StackLayout gap={1} direction="row" align="center">
+          <CreditCardIcon size={2} />
+          <H3>Credit Card</H3>
+        </StackLayout>
+        <Text>Link credit card to your payments account.</Text>
+      </StackLayout>
+    </InteractableCard>
+    <InteractableCard {...args} style={{ width: "180px" }} disabled>
+      <StackLayout gap={1}>
+        <StackLayout gap={1} direction="row" align="center">
+          <BankIcon size={2} />
+          <H3 disabled>Bank wire</H3>
+        </StackLayout>
+        <Text disabled>Link your bank to your payments account.</Text>
+      </StackLayout>
+    </InteractableCard>
+    <InteractableCard {...args} value="crypto" style={{ width: "180px" }}>
+      <StackLayout gap={1}>
+        <StackLayout gap={1} direction="row" align="center">
+          <DiamondIcon size={2} />
+          <H3>Cryptocurrency</H3>
+        </StackLayout>
+        <Text>Link your crypto to your payments account.</Text>
+      </StackLayout>
+    </InteractableCard>
+  </InteractableCardGroup>
+);
+
+export const InteractableCardGroupRadio: StoryFn<typeof InteractableCard> = (
+  args
+) => {
+  const [selected, setSelected] = useState<InteractableCardValue>();
+
+  return (
+    <StackLayout>
+      <StackLayout gap={1}>
+        <H3>Payment method</H3>
+        <Text>Choose your payment method from the options below.</Text>
+      </StackLayout>
+      <InteractableCardGroup
+        onChange={(_event, value) => {
```

Comment:
is this api okay? having to use event and value? I was thinking if we changed it to input then we might not need to do this